substrate-archive-backend/src/database.rs

```diff
@@ -136,7 +143,7 @@ impl<H: Clone> DatabaseTrait<H> for ReadOnlyDatabase {
 }
 
 /*
-impl KeyValueDB for ReadOnlyDatabase {
+impl KeyValueDB for ReadOnlyDB {
```

Comment:
There's some "uncommented code` here, why? Can it be removed?

Reply:
It can be removed, something I left in the code from a while back